### PR TITLE
Add tools

### PR DESCRIPTION
**Pull request by irisdianauy**
Added the following tools to bulk-rnaseq.yaml:
- fastq_quality_trimmer (fastx-toolkit)
- fastq_pair

### CODE REVIEW

File: bulk-rnaseq.yaml
Comment (pinin4fjords) [inline bulk-rnaseq.yaml:126]: These are generic fastq handling tools rather than necessarily RNA-seq specific. Could you maybe a identify a more generic categorisation in the UI? The same might go for the tools above.

Comment (irisdianauy) [reply bulk-rnaseq.yaml:126]: Done in 02cf0096eb7d8087f6afdeef12bf7454f7f23822, but requires that [changes to atlas-lsf-ebi-ansible repo](https://gitlab.ebi.ac.uk/ebi-gene-expression/galaxy-lsf-ebi-ansible/-/merge_requests/8) be merged for new tool panel section names to become valid.

**Review by pinin4fjords — APPROVED** [review]
Looks good, thanks for addressing the comments

**Comment by irisdianauy** [comment]
Thanks, @pinin4fjords.  I'll merge this once the new tool panel sections in codon galaxy are valid.

**Comment by pcm32** [comment]
The changes we were waiting for have been merged and applied to both prod and dev galaxy instances, so will merge.

**Comment by irisdianauy** [comment]
Thank you, @pcm32!

**Pull request merged**
